specs/patterns.md

```diff
@@ -518,3 +519,210 @@ The following links are actual examples already using this pattern, but not 100
+  }]
+}
+```
+
+
+### Specification
+
+A JSON Data Resource MUST be a [Data Resource][dr], that is it MUST conform to the [Data Resource specification][dr].
+
+In addition:
+
+* The Data Resource `schema` property MUST follow the [JSON Schema](https://json-schema.org/) specification,
+  either as a JSON object directly under the property, or a string referencing another
+  JSON document containing the JSON Schema
+- There `MUST` be a `profile` property with the value `json-data-resource`
+* The data the Data Resource describes MUST, if non-inline, be a JSON file
+
+
+### JSON file requirements
+
+When `"format": "json"`, files must strictly follow the [JSON specification](https://www.json.org/). Some implementations `MAY` support `"format": "jsonc"`, allowing for non-standard single line and block comments (`//` and `/* */` respectively).
+
+### Implementations
+
+None known.
+
+
+## Describing Data Package Catalogs using the Data Package Format
+
+### Overview
+
+There are scenarios where one needs to describe a collection of data packages, such as when building an online registry, or when building a pipeline that ingests multiple datasets.
+
+In these scenarios, the collection can be described using a "Catalog", where each dataset is represented as a single resource which has:
+```json
+{
+    "profile": "data-package",
+    "format": "json"
+}
+```
+
+### Specification
+The Data Package Catalog builds directly on the Data Package specification. Thus a Data Package Catalog `MUST` be a Data Package and conform to the [Data Package specification][dp].
+
+The Data Package Catalog has the following requirements over and above those imposed by Data Package:
+* There `MUST` be a `profile` property with the value `data-package-catalog`, or a `profile` that extends it
+* Each resource `MUST` also be a Data Package
+
+#### Examples
+A generic package catalog:
+```json5
+{
+  "profile": "data-package-catalog",
+  "name": "climate-change-packages",
+  "resources": [
+    {
+      "profile": "json-data-package",
+      "format": "json",
+      "name": "beacon-network-description",
+      "path": "https://http://beacon.berkeley.edu/hypothetical_deployment_description.json"
+    },
+    {
+      "profile": "tabular-data-package",
+      "format": "json",
+      "path": "https://pkgstore.datahub.io/core/co2-ppm/10/datapackage.json"
+    },
+    {
+      "profile": "tabular-data-package",
+      "name": "co2-fossil-global",
+      "format": "json",
+      "path": "https://pkgstore.datahub.io/core/co2-fossil-global/11/datapackage.json"
+    }
+  ]
+}
+```
+
+A minimal tabular data catalog:
+```json5
+{
+  "profile": "tabular-data-package-catalog",
+  "name": "datahub-climate-change-packages",
+  "resources": [
+    {
+      "path": "https://pkgstore.datahub.io/core/co2-ppm/10/datapackage.json"
+    },
+    {
+      "name": "co2-fossil-global",
+      "path": "https://pkgstore.datahub.io/core/co2-fossil-global/11/datapackage.json"
+    }
+  ]
+}
+```
```

Comment:
What about examples where the data package json is inlined (is that possible)? That would be interesting ... (but may not be possible ...).

Reply:
@rufuspollock seems like it should be fine to me, but I might be missing some kind of conflict? Added an example of what it would look like.